driver-core/src/main/java/com/datastax/driver/core/Connection.java

```diff
@@ -972,7 +1065,7 @@ public void run(Timeout timeout) {
         public void handle(Message.Response response);
     }
 
-    private static class PipelineFactory implements ChannelPipelineFactory {
+    private static class ChannelInitializer extends io.netty.channel.ChannelInitializer<SocketChannel> {
```

Comment:
Actually I also named the child class ChannelInitializer, which is probably not a great idea in the first place.